Source/Csla/Csla.csproj

```diff
@@ -12,6 +12,16 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>CslaKey.snk</AssemblyOriginatorKeyFile>
     <BaseOutputPath>..\..\Bin</BaseOutputPath>
+    <EnableTrimAnalyzer Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">true</EnableTrimAnalyzer>
```

Comment:
You are checking against net7.0 here. But we don't support net7.0 explicitly anymore. We are only building against 6,8 and 9 iirc

Reply:
Only net8 and 9

Reply:
Okay. I’m probably overthinking things